user/example-models/oracle/oracle_with_boolean.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+
+from h1st.model.fuzzy import (
+    FuzzyModeler,
+    FuzzyRules,
+    FuzzyVariables,
+    FuzzyMembership as fm,
+)
+from h1st.model.oracle import OracleModeler
+
+
+def build_fuzzy_model():
```

Comment:
Why is this `build_fuzzy_model()` while the example is `oracle_with_boolean`? Make sure the name is consistent, at least not contradictory or misleading.